### ISSUE TITLE
Add a button to copy the lyrics to clipboard

### ISSUE THREAD

**Issue by swapnilraj** [JorikSchellekens/lyricist#3]

**Comment by y1n0** [comment]
why don't you just highlight them and press ctrl+c? it is not hard to do that on a computer.

**Comment by swapnilraj** [comment]
Agreed, but in my opinion there is not harm in adding a small feature like this, it makes copying the lyrics even easier.
